src/platform/common/scriptPathUtils.ts

```diff
@@ -0,0 +1,55 @@
+import { inject } from 'inversify';
+import * as uriPath from '../vscode-path/resources';
+import { IExtensionContext, IScriptPathUtils } from './types';
+
+export class ScriptPathUtils implements IScriptPathUtils {
+    constructor(@inject(IExtensionContext) private readonly context: IExtensionContext) {}
+
+    get dataFrameLoading() {
+        const sysPath = uriPath.joinPath(
+            this.context.extensionUri,
+            'pythonFiles',
+            'vscode_datascience_helpers',
+            'dataframes'
+        );
+        const dataFrameImport = `__import__('vscodeDataFrame')`;
+        return {
+            dataFrameSysImport: `import sys\nsys.path.append("${sysPath.path.replace(/\\/g, '\\\\')}")`,
```

Comment:
To make `scriptPathUtils` less Node specific, I made `scriptPath` into a `Uri`. However, In the current design I can't fix `dataFrameSysImport` to either use `fsPath` or `path` depending on the environment. In a future commit, I could change the design to, for example, expose both `dataFrameSysImport` and `dataFrameSysImportFsPath`, as a possible solution. However, I should ask: Would `path` here work both for Node.js and the browser?

Reply:
cc: @DonJayamanne

Reply:
@rchiodo will the use of `path` work here?
Since we're dealing with remote jupyter kernels, the values sent to the kernel should contain actual file paths, hence I'm not sure `path` property would work here. I thought you mentioned that its not a real path and has some funky logic to escape drive letters & network paths.

I.e. I don't think this will work in desktop & remote. Possible this is an exception where we can use fsPath or we need something similar.

Reply:
Wow, good catch. Yeah this needs to be redesigned entirely for the web. It needs to import the entire script as a text file and run it that way instead of importing it through the file path.

Reply:
We could do like I did for the addRunCellHook.

Reply:
> Maybe you could move the reading of scripts into scriptPathUtils? Then it's isolated in one spot.

Feels like we'll have to address the above comment in this PR, moving reading of the script file to this new scriptPathUtils file and then importing the script as mentioned by @rchiodo

Reply:
> Looks like there are some build errors? But the approach looks good to me

I sure do hope the build failures are a result of the use of `path`, if not, I'd like some tests for this (perhaps we can add them in debt week - to ensure we don't introduce such breakages)